parallel_cluster/conf/pcluster_client.env.yml

```diff
@@ -4,6 +4,9 @@ dependencies:
   - pip
   - boto3
   - conda-forge::jq
+  # FIXME this install awscli v1.18
```

Comment:
True, this should be removed and superseeded by EC2 Image Builder component `awscliv2` (or amazon-linux-extras equivalent if there's one), making it system-wide instead of conda/user dependant.

Reply:
I think that's possible, with conda env-vars.sh they can prepend aws2 to their path

Reply:
I've got this working by installing just pip and aws-parallel-cluster through pip. Do we need boto3 and jq and awscli?

Reply:
Yes, I bet users will be needing all of those, so it'd be nice to have them pre-installed on the AMI/base system, IMHO.

Reply:
So we wait for AWS to create v2 of their pip installation? Should we expect that to be coming soon?

Reply:
Is it tricky/messy to install v2 from our EC2-Image-Builder components?

Reply:
I thought this the virtualenv installation for the local user to then run pcluster - from https://github.com/aws/aws-cli/issues/4947 it seems clear that they're not going to update it. Comments from users are pretty brutal.